services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/app/AlcorControllerApp.java

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.dataplane.app;
+
+import com.futurewei.alcor.dataplane.app.onebox.OneBoxConfig;
+import com.futurewei.alcor.dataplane.app.onebox.OneBoxUtil;
+import com.futurewei.alcor.dataplane.db.redis.RedisConfiguration;
+import com.futurewei.alcor.dataplane.logging.Logger;
+import com.futurewei.alcor.dataplane.logging.LoggerFactory;
+import com.futurewei.alcor.dataplane.model.HostInfo;
+import com.futurewei.alcor.dataplane.resourcemgr.physical.nodemgmt.DataCenterConfig;
+import com.futurewei.alcor.dataplane.resourcemgr.physical.nodemgmt.DataCenterConfigLoader;
+import com.futurewei.alcor.dataplane.resourcemgr.physical.nodemgmt.NodeManager;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+
+@SpringBootApplication(scanBasePackages = "com.futurewei.alcor.dataplane")
+@Import({RedisConfiguration.class})
+public class AlcorControllerApp {
+
+    public static void main(String[] args) {
+        //Class<?>[] sources = {Alcor.class, RedisConfiguration.class};
+        SpringApplication.run(AlcorControllerApp.class, args);
```

Comment:
This is the legacy code. Let us make the application code simple.

Reply:
fixed in 4b41302e31fcf34d092f187c983d732c7adafecf

Reply:
DP service is just one of many microservices in the controller. Why would you name DP manager as "AlocrControllerApp"?

Reply:
fixed in 136f4b5